client/assets/pages/_events-page.scss

```diff
@@ -540,3 +540,14 @@ main {
   text-transform: capitalize;
   margin-bottom: 5px;
 }
+
+.category-label {
+  font-size: .9375rem;
```

Comment:
It makes sense to specify this in full `0.9375rem`, I've had one crazy bug in the past where something defined this way didn't work in a particular browser or something. Can't remember the exact scenario but it got fixed by specifying this in full.